### PR TITLE
PyPy 5.3.0

### PR DESCRIPTION
**Pull request by chuanconggao**
- [X] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [X] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [X] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

**Comment by jenshnielsen** [comment]
Any particular reason this was closed?

**Comment by chuanconggao** [comment]
There are some build errors for this pull request. The compiling for building bottles takes forever to finish right now. I am still trying to figure it out.

**Comment by cclauss** [comment]
I think this issue should remain open.  Pypy builds talk a long time but what you have here looks right to me.

**Comment by DomT4** [comment]
Oh, heh. When this happens in future (i.e. the build times out rather than "fails") feel free to @ one of us maintainers and nudge us to bump the timeout for you. It's back in the queue, with 3 hours to build instead of the default timeout.

File: Formula/pypy.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Pypy < Formula
   desc "Highly performant implementation of Python 2 in Python"
   homepage "http://pypy.org/"
-  url "https://bitbucket.org/pypy/pypy/downloads/pypy-5.1.1-src.tar.bz2"
-  sha256 "ca3d943d7fbd78bb957ee9e5833ada4bb8506ac99a41b7628790e286a65ed2be"
+  url "https://bitbucket.org/pypy/pypy/downloads/pypy2-v5.3.0-src.tar.bz2"
+  sha256 "4142eb8f403810bc88a4911792bb5a502e152df95806e33e69050c828cd160d5"
   revision 1
```
Comment (DomT4) [inline Formula/pypy.rb]: Oh, you will need to remove this line though. Formula `revision`s get wiped on update.

**Comment by chuanconggao** [comment]
@DomT4 Thanks for the help. Now it has passed the checks. BTW, I have also removed the `revision` line.

**Comment by MikeMcQuaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Comment by cclauss** [comment]
Nice teamwork here folks!  @DomT4, @MikeMcQuaid Could there be an automated way for the build process to _always_ give Pypy, and Pypy3 builds more time to complete than other builds?  If not, this issue will recur on each new Pypy release.

**Comment by MikeMcQuaid** [comment]
@cclauss I don't think Jenkins provides such a thing, unfortunately.

**Pull request closed without merging**